Extensions/retags.js

```diff
@@ -265,7 +265,7 @@ XKit.extensions.retags = {
 	$('<style class="retags"> ' +
 		'.ui_note, .ui_notes .activity-notification { display: none; } ' +
 		'.ui_note.is_retags, .ui_note.is_reply, .ui_note.is_response, .ui_note.is_user_mention { display: block; } ' +
-		'.activity-notification.is_retags, .activity-notification.is_reply, .activity-notification.is_reblog:not(.naked), .activity-notification.user_mention, .activity-notification.note_mention { display: flex; }' +
+		'.activity-notification.is_retags, .activity-notification.is_reply, .activity-notification.is_reblog, .activity-notification.user_mention, .activity-notification.note_mention { display: flex; }' +
```

Comment:
if `.naked` doesn't exist on the new activity page I can't imagine this is necessary; shows all reblogs as retags/responses for me